### ISSUE TITLE
Link not working

### ISSUE THREAD

**Issue by ss8319** [BCV-Uniandes/SIMBA#3]
Hi, the link attached in this repository is not working such as access to homepage, pretrained models and the dataset to train SIMBA. Can you share the correct one?
http://marr.uniandes.edu.co/baar/home
http://marr.uniandes.edu.co/SIMBA/pretrained_models/model_5.pth
http://marr.uniandes.edu.co/baar/dataset

**Comment by mc-escobar11** [comment]
Thank you for your interest on the RHPE dataset, our webpage has been going through some modifications and is not available at the moment. However, you can access the datasets and annotations through the following links. If you need to evaluate the performance on the test set please send me an email with a csv file following the same format of the examples ([RHPE](https://drive.google.com/file/d/1tAQW81d3PHmvrS1wPhGmmZZvyS5DrTuK/view?usp=sharing), [RSNA](https://drive.google.com/file/d/1vGF9BtXhzXlAf9Mq3jwEBmqi6JZx6uZf/view?usp=sharing)).

RHPE: [Readme](https://drive.google.com/file/d/1hxAV5trnxTbIxJwoH3UsSxPsSywpWRrF/view?usp=sharing)| [Train](https://drive.google.com/file/d/1Lq3XCOdOkJJKXPVPEYLvf1ScaFGh5uU6/view?usp=sharing)| [Validation](https://drive.google.com/file/d/1bjvC1YZ0h0TK1MXeamWXb8dF6wx-ZKQQ/view?usp=sharing)| [Test](https://drive.google.com/file/d/1sNFFDxbYjIeeBoSeYUokyq4dts_ZgnA_/view?usp=sharing)| [Annotations](https://drive.google.com/file/d/1NOtsunlmnYMufXEeeIHO_DPI77HJzx31/view?usp=sharing)
RSNA:[Train](https://drive.google.com/file/d/1SKmjs8Lvjv8DBs3LSy0XkbWDyJr_U3YF/view?usp=sharing)| [Validation](https://drive.google.com/file/d/17JM6Y5VeVgS35ywTKJ1U5CqHhDsF8irf/view?usp=sharing)| [Test](https://drive.google.com/file/d/1-8akTtn8rAf4cQklIJQ1NUbFnTlGocgc/view?usp=sharing)| [Annotations](https://drive.google.com/file/d/1mOveYqmsiiONFxAUhDCWaFvCnq6GSIRl/view?usp=sharing)

Please let me know if you have any other questions, I will try to reply as soon as possible.

**Comment by ss8319** [comment]
Thanks so much @mc-escobar11. Is it possible to share links to the trained model weights.

**Comment by henryzhao0615** [comment]
Hi @mc-escobar11 ,
May I ask where can I find the ground truth bone age annotations for test set. Thanks!

**Comment by mc-escobar11** [comment]
Hi @henryzhao0615, You can find the images for the test set in the  [Bone Age Assessment Resources](https://bcv-uniandes.github.io/baar-wp/) and if you want to evaluate a method you can do it in the [public server](https://eval.ai/web/challenges/challenge-page/2297/overview). We do not release the test annotations to keep the task fair and consistent across methods.
